lib/PuppeteerSharp/Dialog.cs

```diff
@@ -9,11 +10,13 @@ namespace PuppeteerSharp
     /// <example>
     /// An example of using Dialog class:
     ///<code>
+    ///<![CDATA[
     /// Page.Dialog += async (sender, e) =>
     /// {
     ///     await e.Dialog.Accept();
     /// }
-    /// await Page.EvaluateExpressionAsync("alert('yo');");
+    /// await Page.EvaluateExpressionAsync<object>("alert('yo');");
```

Comment:
`await Page.EvaluateExpressionAsync("alert('yo');");` will still be valid right?.
I'd like to keep it that way. An EvaluateExpressionAsync with no generic type would express a "void" expression.

Reply:
Yes, I'll correct this comment